### ISSUE TITLE
[Nightly Issue] Object Pooling Causes Desync

### ISSUE THREAD

**Issue by broken-usb** [AdalynBlack/Loadstone#20]
Having object pooling enabled will cause certain objects position to be different from other players(that don't have the mod), most noticeable on the Manor/Mansion where the bookshelves you see aren't in the same spot where other players see them which can and will result in players walking inside of them for you and vice-versa.

Pretty sure the same happens on the Facility interior but i haven't tested it.

**Comment by AdalynBlack** [comment]
Yes, this is a known issue, and is why object pooling is marked as experimental and nightly only
The feature is highly developmental and not ready for full-time use yet
